src/installation/guides/fde.md

```diff
@@ -3,6 +3,13 @@
 **Warning**: Your drive's block device and other information may be different,
 so make sure it is correct.
 
+## Caution
+
+Filesystem-level encryption with AEAD is more reliable than using
+[XTS](https://sockpuppet.org/blog/2014/04/30/you-dont-want-xts/),
```

Comment:
The article you linked mentions:

> Be wary of applications that claim to use it for anything other than disk encryption.

We are using it for disk encryption.

> If you’re an end-user looking for crypto advice: use Truecrypt, use Filevault, use dm-crypt.

LUKS uses `dm-crypt` as its backend, so we are also doing things correctly here.